partcad/src/partcad/ai_google.py

```
@@ -21,48 +21,72 @@
-            google_api_core_exceptions = import_optional("google.api_core.exceptions", "Google AI", "ai-google")
+        # `google.api_core.exceptions` is not used any more: the new SDK raises
+        # its own `google.genai.errors.APIError` hierarchy.
+        if google_genai_errors is None:
+            google_genai_errors = import_optional("google.genai.errors", "Google AI", "ai-google")
 
         latest_key = user_config.google_api_key
         if not latest_key:
             latest_key = interactive.prompt("API_KEY_GOOGLE", "Enter Google Cloud Generative Language API key")
         if latest_key != GOOGLE_API_KEY:
             GOOGLE_API_KEY = latest_key
             if not GOOGLE_API_KEY is None:
```

Ruff (0.15.21)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)